roottest/root/meta/Event.h

```diff
@@ -1,203 +0,0 @@
-#ifndef ROOT_Event
```

Comment:
Fine to replace with `roottest/root/io/event`.  The only significant difference are:
```
40,41c43,44
<    Double32_t   fCharge;       //Charge of this track
<    Double32_t   fVertex[3];    //Track vertex position
---
>    Double32_t   fCharge;       //[-1,1,2] Charge of this track
>    Double32_t   fVertex[3];    //[-30,30,16] Track vertex position
45c48
<    Double32_t*  fPointValue;   //[fNsp] a special quantity for some point.
---
>    Double32_t*  fPointValue;   //[fNsp][0,3] a special quantity for some point.
```
where the customized `Double32_t` are actually improve test coverage per se.